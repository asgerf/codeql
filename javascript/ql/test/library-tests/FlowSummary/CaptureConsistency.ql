import semmle.javascript.dataflow2.VariableCaptureSpecific::VariableCaptureOutput::ConsistencyChecks
