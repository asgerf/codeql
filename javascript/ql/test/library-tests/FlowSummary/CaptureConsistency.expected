uniqueToString
uniqueEnclosingCallable
uniqueDominator
localDominator
localSuccessor
uniqueDefiningScope
variableIsCaptured
uniqueLocation
uniqueCfgNode
| tst.js:1:13:1:13 | t | Expr has no cfg node |
uniqueWriteTarget
| VariableWrite has no target variable |
uniqueWriteSource
| VariableWrite has no source expression |
uniqueWriteCfgNode
| VariableWrite has no cfg node |
localWriteStep
uniqueReadVariable
closureMustHaveBody
closureAliasMustBeLocal
variableAccessAstNesting
uniqueCallableLocation
consistencyOverview
| Expr has no cfg node | 1 |
| VariableWrite has no cfg node | 2 |
| VariableWrite has no source expression | 2 |
| VariableWrite has no target variable | 48 |
